docs/source/examples/relaxation_output.txt

```diff
@@ -45,7 +45,8 @@ With constraints relaxed individually
   # minimize
         C_Relax.1_(0,)*C_Relax.1_(1,)*x**0.01
   # subject to
-        C_Relax.1 >= [gpkit.Monomial(x*x_max**-1), gpkit.Monomial(x**-1*x_min)]
+        x <= C_Relax.1_(0,)*x_max
```

Comment:
this is probably better than the old output, but worth looking at the difference